hi-ml-multimodal/README.md

```diff
@@ -1,11 +1,71 @@
-# Multimodal
+# HI-ML Multimodal Toolbox
 
-## Developer setup
+This toolbox provides models for working with multi-modal health data.
+The code is available on [GitHub][1] and [Hugging Face 🤗][6].
 
-Make file commands:
+## Getting started
 
-* `make pip_local` to install the `hi-ml-multimodal` package in editable mode. This must happen before running tests.
-* `make build` to build the package
-* `make mypy` to run `mypy`
-* `make check` to run `flake8`, `mypy` and `pyright`
-* `make clean` to clean up all temporary files and folders
+The best way to get started is by running the [phrase grounding notebook][2].
+All the dependencies will be installed by the notebook, so Python 3.7 and [Jupyter][3] are the only requirements to get started.
+
+The notebook can also be run on [Binder][4], without the need to download any code or install any libraries:
+
+[![Binder](https://mybinder.org/badge_logo.svg)][4]
+
+## Installation
+
+The latest version from can be installed using `pip`:
+
+```console
+pip install "git+https://github.com/microsoft/hi-ml.git#subdirectory=hi-ml-multimodal"
+```
+
+### Development
+
+For development, it is recommended to clone the repository and set up the environment using [`conda`][5]:
```

Comment:
Do we require the code to run on a machine with GPU? I don't remember us testing the notebook on CPU.

Reply:
The CI runs on CPU. Same for Binder:

[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/microsoft/hi-ml/HEAD?labpath=hi-ml-multimodal%2Fnotebooks%2Fphrase_grounding.ipynb)

So we should be good.